### PR TITLE
Updated push.is-a.dev

### PR DESCRIPTION
**Pull request by 7x9z**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

[https://pushingcc.github.io/](url)

### CODE REVIEW

File: domains/push.is-a.dev.json
Comment (Stef-00012) [inline domains/push.is-a.dev.json:1]: you should remove `.is-a.dev` from file name

File: domains/_discord.push.is-a.dev.json
Comment (Stef-00012) [inline domains/_discord.push.is-a.dev.json:1]: you should remove `.is-a.dev` from file name

**Review by Stef-DP — DISMISSED** [review]

File: domains/push.is-a.dev.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal Page / File storage",
+    "repo": "https://github.com/pushingcc/pushingcc.github.io",
+    "owner": {
+        "username": "pushingcc",
+        "email": "pushing@mailfence.com",
+        "twitter": "<N/A>"
```
Comment (Stef-00012) [inline domains/push.is-a.dev.json:7]: remove twitter field if you don't have it

**Comment by pushingcc** [comment]
Changes have been added and submitted.

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**